npm/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ologbonowiwi/tailcall",
+  "version": "0.1.0",
```

Comment:
This version should be taken from release.

Reply:
I am doing it on `lib/index.js`. I'm letting this here as a reference, but it'll override the prepare script.